syftlib/src/main/java/org/openmined/syft/Syft.kt

```diff
@@ -19,6 +19,15 @@ import java.util.concurrent.atomic.AtomicBoolean
 
 private const val TAG = "Syft"
 
+/**
+ * The main Worker Class handling all the Federated Training Cycles. 
+ * We recommend android devices to not create multiple jobs as usually they are resource-intensive and may hamper user experience.
+ * @see SyftConfiguration contains the allowed configuration parameters for syft worker. @see syftconfiguration.
```

Comment:
it should be @param syftConfiguration . [configuration][SyftConfiguration] remove the last @see